.travis.yml

```diff
@@ -51,14 +51,15 @@ jobs:
   include:
     - stage: baseline builds
       os: linux
-      compiler: gcc
-      env: GCOV=gcov CMAKE_FLAGS="$CMAKE_FLAGS -DUSE_GCOV=ON"
-    - os: linux
-      compiler: gcc
+      compiler: clang-4.0
+      # Use Lua so that ASAN can test our embedded Lua support. 8fec4d53d0f6
       env: >
-        FUNCTIONALTEST=functionaltest-lua
+        CLANG_SANITIZER=ASAN_UBSAN
```

Comment:
this section should go back exactly where it was before all of these changes, e.g. in 491a3210a9b8e67fc91817659fda27585a25e4e1 , so that diffing this file with older versions does not have unnecessary variations.